packages/material-ui-codemod/README.md

```diff
@@ -1067,6 +1068,67 @@ npx @material-ui/codemod@next v5.0.0/with-width <path>
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#material-ui-core-styles).
 
+#### `mui-replace`
+
+Replace every occurrence of `material-ui` related package with the the new package name (listed below) except these packages (`@material-ui/pickers`, `@material-ui/data-grid`, `@material-ui/x-grid` & `@material-ui/x-grid-data-generator`)
```

Comment:
First-order feedback:

```suggestion
Replace every occurrence of `material-ui` related package with the new package names (listed below) except these packages (`@material-ui/pickers`, `@material-ui/data-grid`, `@material-ui/x-grid` & `@material-ui/x-grid-data-generator`).
```

Second-order feedback: Maybe use this as an opportunity to frame the MUI Core vs. MUI X separation?

```suggestion
Replace every occurrence of the MUI Core packages with the new package names (listed below). `@material-ui/pickers`, `@material-ui/data-grid`, `@material-ui/x-grid` & `@material-ui/x-grid-data-generator` are not part of MUI Core.
```